server/app/views/admin/programs/ProgramBlockAllAspectsView.java

```diff
@@ -243,13 +265,23 @@ private DivTag renderBlockList(
                   "border",
                   "border-white",
                   StyleUtils.hover("border-gray-300"),
-                  selectedClasses)
-              .with(
-                  a().withClasses("flex-grow", "overflow-hidden")
-                      .withHref(editBlockLink)
-                      .with(p(blockName), p(questionCountText).withClasses("text-sm")))
-              .with(moveButtons);
+                  selectedClasses);
 
+      // TODO(#3162) use a different route when viewAllowsEditingProgram is
+      //  false once a read only navigation option is available.
+      String switchBlockLink =
```

Comment:
Should this be in the `if viewAllowsEditingProgram()` check since it's the DRAFT route?

Reply:
Can this comment be closed?